File: Babbar/reverse a list.cpp

void swap(char *a, char *b){
    char *t;
    t = a;a = b;b=t;
}

string reverseWord(string str){
    int n = str.length();
    for(int i = 0; i < n / 2 ; i++){
        swap(str[i],str[n - 1 - i ]);
    }
    return str;
  //Your code here
}

File: Babbar/reverse a list.py
def reverseWord(s):
    #your code here
    i,j = 0,len(s) - 1 
    s = list(s)
    while i < j:
        s[i],s[j] = s[j],s[i]
        i += 1
        j-= 1
    ans = ""
    for i in s:
        ans += i
    return ans